packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.scss

```diff
@@ -2,49 +2,53 @@
 @import '../../shared/styles/variables';
 
 .container {
-	@include main;
-	display: flex;
-	height: 100%;
-	flex-direction: column;
-}
-
-.calendar-row {
-	display: flex;
-}
-
-.calendar-item {
-	display: flex;
-	flex: 1;
-	justify-content: center;
-	align-content: center;
+	width: 100%;
+	position: relative;
 }
 
 .calendar-header {
-	.calendar-row {
+	&:after {
+		content: '';
+		position: absolute;
+		width: calc(100% - 2rem);
+		top: 2rem;
+		left: 1rem;
+		border-bottom: $border-default;
+	}
+
+	th {
```

Comment:
Ask Frassiner for this one, but if you can i think we have to avoid elemnt selector.

Reply:
@frassinier: any suggestion ?
Here the th is scoped to `.calendar-header`. I just didn't want to add a classname on each node